dist/rpm/singularity-ce.spec.in

```diff
@@ -115,10 +123,10 @@ container platform designed to be simple, fast, and secure.
         --mandir=%{_mandir} \
         --infodir=%{_infodir}
 
-%make_build -C builddir old_config= V=
+make -C builddir old_config= V=
```

Comment:
I wonder if there was any utility in using these macros to begin with? And if there was, whether we can just add a block that defines them by brute force to `make` if we're on the relevant SLES versions.